package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-application",
-  "version": "2.4.3",
+  "version": "2.4.4",
```

Comment:
I think a new database table and whole new API merits a minor version increase, rather than a micro one. If we were using semantic versioning based on commit messages, this would be a feat: and merit minor increase